apps/researcher/src/messages/en/messages.json

```diff
@@ -161,7 +162,9 @@
     "membershipCount": "{count, plural, =0 {0 members} =1 {1 member} other {# members}}",
     "objectListCount": "{count, plural, =0 {0 lists} =1 {1 list} other {# lists}}",
     "objectListCountError": "Error",
-    "searchPlaceholder": "Search for community"
+    "searchPlaceholder": "Search for community",
+    "addCommunity": "Create a community",
+    "toggleMyCommunities": "Show only my communities"
```

Comment:
Or rename `toggleMyCommunities` to `showMyCommunities` (to make its meaning clearer)?